### PR TITLE
Added options remaining issuance to spreadsheet

### PR DESCRIPTION
**Pull request by daviidy**
# ASK
Issue #18

### CODE REVIEW

File: src/workbook/stakeholder-sheet/holdings-columns.ts [hunk 1]
```diff
@@ -38,6 +39,8 @@ export class StakeholderColumn {
     myColumn
       .createNestedRange()
       .addBlankCell(Styles.default)
+      .addCell("Options Remaining for Issuance")
+      .addBlankCell(Styles.default)
```
Comment (pjohnmeyer) [inline src/workbook/stakeholder-sheet/holdings-columns.ts:43]: **Non-blocking:** If you put the `{style: Styles.default}` as a parameter to the nested range, you shouldn't need to pass it on LL41-43

File: src/workbook/stakeholder-sheet/holdings-columns.ts [hunk 2]
```diff
@@ -57,6 +60,8 @@ export class StakeholderGroupColumn {
       .addCell("Stakeholder Group", Styles.subheader)
       .addBlankCells(stakeholders.length, Styles.default)
       .addBlankCell(Styles.default)
+      .addBlankCell(Styles.default)
+      .addBlankCell(Styles.default)
```
Comment (pjohnmeyer) [inline src/workbook/stakeholder-sheet/holdings-columns.ts:64]: **Non-blocking:** Use `.addBlankCells` instead of multiple `.addBlankCell` calls.

**Review by pjohnmeyer — CHANGES_REQUESTED** [review]

File: src/workbook/stakeholder-sheet/holdings-columns.ts [hunk 3]
```diff
@@ -184,20 +193,32 @@ export class StockPlanColumn {
...
 
+    const initialSharesReserved =
+      stockPlan.initial_shares_reserved !== undefined
+        ? stockPlan.initial_shares_reserved
+        : 0;
+    const options_remaining =
+      initialSharesReserved !== 0
+        ? new Big(initialSharesReserved).toNumber() - total_holdings
+        : 0;
+
```
[the remark this answers is not in the document]
Comment (daviidy) [reply src/workbook/stakeholder-sheet/holdings-columns.ts:216]: One of the criteria we can look at is whether or not we need to consume objects from the package in order to do the calculations. If we need to consume objects we can go with the model.
But as shown in issue #19, we can rely on the values in the spreadsheet.

[the remark this answers is not in the document]
Comment (pjohnmeyer) [reply src/workbook/stakeholder-sheet/holdings-columns.ts:216]: Yes, in the case of #19 there are no "in-tool" calculations -- other than assembling the formula. That definitely belongs in the workbook code b/c only the workbook knows where the cells are.

We can discuss the other cases next week when I'm not on-call; I'm planning on spending part of my Tuesday documenting some things in OCX.

**Review by pjohnmeyer — APPROVED** [review]

**Pull request merged**